logic/src/commonTest/kotlin/com/wire/kalium/logic/sync/receiver/FeatureConfigEventReceiverTest.kt

```diff
@@ -270,7 +279,7 @@ class FeatureConfigEventReceiverTest {
             verify(arrangement.userConfigRepository)
                 .suspendFunction(arrangement.userConfigRepository::setTeamSettingsSelfDeletionStatus)
                 .with(matching {
-                    it.hasFeatureChanged == true && it.enforcedSelfDeletionTimer !is TeamSelfDeleteTimer.Disabled
+                    it.hasFeatureChanged == false && it.enforcedSelfDeletionTimer !is TeamSelfDeleteTimer.Disabled
```

Comment:
@gongracr I changed the expectation of `hasFeatureChanged` to `false`. It doesn't make sense to me to flag the feature has having changed if we didn't have any previous persisted configuration of it because that would imply that we would flag the feature as changed after the initial sync.